mots.csl

```diff
@@ -0,0 +1,299 @@
+        <text variable="volume" prefix=", n° " suffix=","/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="lieu-edition">
+    <choose>
+      <if type="book chapter thesis report paper-conference" match="any">
+        <group delimiter=", ">
+          <text variable="publisher-place" form="short"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="editeur">
+    <choose>
+      <if type="book chapter thesis report paper-conference" match="any">
+        <group delimiter=", ">
+          <text variable="publisher" form="short"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publication">
+    <choose>
+      <if type="thesis">
+        <group>
+          <text value="Thèse de doctorat soutenue à "/>
+          <text macro="lieu-edition"/>
+          <text macro="date-publication" prefix=", "/>
+        </group>
+      </if>
+      <else>
+        <group delimiter=", ">
+          <text macro="lieu-edition"/>
+          <text macro="editeur"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages-chapitre">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text variable="volume" suffix=" :"/>
+        <text variable="page" prefix="p. "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="pages-article">
+    <choose>
+      <if type="article-journal">
+        <text variable="page" prefix="p. "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="type-acces">
+    <group delimiter=" ">
+      <text variable="URL" prefix=" "/>
+      <choose>
+        <if variable="URL">
+          <group prefix="(consulté le " suffix=")">
+            <date variable="accessed">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" suffix=" "/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </if>
+      </choose>
+    </group>
+  </macro>
+  <!-- INSTRUCTIONS DE FORMATAGE DES CITATIONS -->
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-names="true" disambiguate-add-year-suffix="true">
+    <layout delimiter=" ; ">
+      <choose>
+        <if position="ibid-with-locator">
+          <group delimiter=", ">
+            <text term="ibid" font-style="italic" suffix="."/>
+            <text variable="locator" prefix="p. "/>
+          </group>
+        </if>
+        <else-if position="ibid">
+          <text term="ibid" font-style="italic" suffix="."/>
+        </else-if>
+        <else>
+          <group delimiter=", ">
+            <text macro="auteurs-court"/>
+            <date variable="issued">
+              <date-part name="year"/>
+            </date>
+            <text macro="indication-localisation"/>
+          </group>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <!-- INSTRUCTIONS DE FORMATAGE DE LA BIBLIOGRAPHIE -->
+  <bibliography entry-spacing="0">
+    <sort>
+      <key macro="contributeurs-tri"/>
```

Comment:
I don't understand the difference between the auteurs macro and this one -- they seem redundant. Why not just sort by `auteurs`?